src/backend/logger/__tests__/logfile.test.ts

```diff
@@ -85,10 +85,7 @@ describe('logger/logfile.ts', () => {
     const data = logfile.createNewLogFileAndClearOldOnces()
 
     expect(logError).toBeCalledWith(
-      [
-        expect.stringContaining('Removing old logs in /tmp/'),
-        Error('unlink failed')
-      ],
+      [expect.stringContaining('Removing old logs in'), Error('unlink failed')],
```

Comment:
is this `/tmp/` removal intentional?

Reply:
it fails on windows if I keep it

Reply:
I can add it back. we should probably use path module so tests aren't expecting linux formatted file paths